### DISCUSSION TITLE
Implementation of SEAL (for link prediction)

### DISCUSSION THREAD

**Discussion by LuisaWerner** [pyg-team/pytorch_geometric#7699] [Ideas]
Hi all, 
does PyG provide an implementation for the method SEAL (https://proceedings.neurips.cc/paper_files/paper/2018/file/53f0d7c537d99b3824f0f99d62ea2428-Paper.pdf)  for link prediction tasks? 
Best, 
Luisa

**Comment by wsad1** [comment]
Check out this example https://github.com/pyg-team/pytorch_geometric/blob/master/examples/seal_link_pred.py.

**Reply by LuisaWerner** [reply]
Thank you!
